src/main/java/uk/co/onsdigital/discovery/model/GeographicArea.java

```diff
@@ -10,7 +10,7 @@
  *
  */
 @Entity
-@Table(name="geographic_area")
+@Table(name="geographic_area", indexes = {@Index(columnList="ext_code")})
```

Comment:
It is not unique. The same ext_code can exist in different hierarchies. We eventually need to know the ext_code and the hierarchy, so the unique constraint will be ext_code + hierarchy ID.